runtime/doc/lsp.txt

```diff
@@ -1653,7 +1653,7 @@ Lua module: vim.lsp.client                                        *lsp-client*
                                 See |Client:notify()|.
       • {cancel_request}        (`fun(self: vim.lsp.Client, id: integer): boolean`)
                                 See |Client:cancel_request()|.
-      • {stop}                  (`fun(self: vim.lsp.Client, force: boolean|integer?)`)
+      • {stop}                  (`fun(self: vim.lsp.Client, timeout: integer|boolean?)`)
```

Comment:
Don't think we should rename this. `force` is more intuitive and meaningful as a dual boolean/integer, vs `timeout`.

Reply:
Done!